tests/CacheCompat/CommonCache.Test.AdalV3/Program.cs

```diff
@@ -19,7 +20,7 @@ public static void Main(string[] args)
         private class AdalV3CacheExecutor : AbstractCacheExecutor
         {
             /// <inheritdoc />
-            protected override async Task<CacheExecutorResults> InternalExecuteAsync(TestInputData testInputData)
+            protected override async Task<List<CacheExecutorAccountResult>> InternalExecuteAsync(TestInputData testInputData)
```

Comment:
You don't seem to need to expose an actual List, as you just iterate through those results, so IEnumerable<CacheExecuteAccountsResukt> should be sufficient.